index.html

```diff
@@ -10,60 +10,167 @@
 </head>
 <body>
```

Comment:
- [OPTIONAL] ⚠️  Your project seems **not responsive**; kindly take time to work on this. It's essential for building platforms that users enjoy using. Here is a [resource](https://www.w3schools.com/html/html_responsive.asp) 🕸️  for your reference. Please work on this to help future reviewers easily approve your projects 🙏🏽